src/map/map.ts

```diff
@@ -207,6 +207,10 @@ export class PropertiesMap {
         };
 
         this._createPlot();
+
+        // Trigger a few updates that are needed to ensure a consistent state
+        this._options.size.property.onchange(this._options.size.property.value, 'JS');
+        this._options.z.property.onchange(this._options.z.property.value, 'JS');
```

Comment:
I think this code should live the applySettings function, but this function should already be called:

The value is updated in applySettings:
https://github.com/cosmo-epfl/chemiscope/blob/d779d1b1d899cb67c5b607a9735809d1d5c23275/src/options.ts#L329

which then call the setter
https://github.com/cosmo-epfl/chemiscope/blob/d779d1b1d899cb67c5b607a9735809d1d5c23275/src/options.ts#L136-L138

which calls `onchange`
https://github.com/cosmo-epfl/chemiscope/blob/d779d1b1d899cb67c5b607a9735809d1d5c23275/src/options.ts#L219

---

I think the actual issue here is that we are applying the saved settings a few lines above in `MapOptions` constructor, before having called `this._connectSettings()`. But we can not connect settings before we have them constructed, so the best fix IMO would be to remove the settings from the MapOption constructor; and apply them after connecting everything.

Reply:
So the problem we were having back then - which I think contributes to this messy design - is that if you connect first, then apply the settings, it triggers a gazillion of redraw calls, and can be *really* slow. 
IMO fixing this requires a separate PR, in which we redesign the whole update mechanism, having a separate "redraw" call, and having an option to update() to trigger the redraw or not (so that if you are changing multiple settings you can redraw just once).

Reply:
Ok, that make sense. In that case I would rather not add a temporary hack to the code and make this PR focus on changing the map settings UI. 

I think we can live for a short while with greyed out boxes, and having them remind us there is something that needs to be fixed, instead of hiding it.

Reply:
I agree with the overarching "update", can we include in the UI a button ala "Apply settings" so that redraw isn't triggered with every change, but rather all at once?